bot/cogs/utility/nickrequest.py

```diff
@@ -21,12 +21,10 @@ def __init__(self, bot: Bot):
     async def nick(self, ctx: commands.Context, *, nick):
         acc_channel = self.bot.get_channel(accept_channel_id)
         nick = str(nick)
-        nick.replace('"', '\"')
-        nick.replace("'", "\'")
 
         # this looks if the request is valid
         for char in nick:
-            if not char in allowed_list:
+            if not char in string.printable:
```

Comment:
Probably doesn't matter at this scale but it'd be better to reuse a `set(string.printable:)` for this.